common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -46,17 +46,21 @@ enum class AntMode { Invalid = 0, ANT_1X1, ANT_2X2, ANT_3X3, ANT_4X4 };
 enum class WiFiChanBW { Invalid = 0, BW_20 = 20, BW_40 = 40, BW_80 = 80 };
 
 struct VAPElement {
+    /**
+     * Basic Service Set (i.e.: VAP name, e.g.: wlan0.0, wlan0.1, wlan0.2, ...).
+     */
+    std::string bss;
```

Comment:
`bss` is used in few meanings and formats along the code:
* Sometimes it is a `string` representing mac address.
* Sometimes it is a sMacAddr representing mac address.
* and here it is the _name_ of the vap.

I think it would be easier for developers, newcomers, maintainers, etc. to have clear names along the code.
I suggest to replace to `vap_name` (or just `vap`) as you explained in the comment above this variable

> (i.e.: VAP name, e.g.: wlan0.0, wlan0.1, wlan0.2, ...)

```suggestion
    std::string vap_name;
```

Reply:
I agree with you that there's room for improvement regarding names and types in this project.
But in this case, I think the field that contains a MAC address (sometimes as a string and others as a sMacAddr) is the BSSID, not the BSS.

The reason to chose `bss` over any other name is simple: the information contained in the `VAPElement` struct is obtained using the hostapd command "STATUS", whose output in a RAX40 is like:

```
...
bss[0]=wlan0
bssid[0]=02:9a:96:fb:59:0f
ssid[0]=dummy_ssid_0
num_sta[0]=0
bss[1]=wlan0.0
bssid[1]=02:9a:96:fb:59:11
ssid[1]=prplMesh
num_sta[1]=0
bss[2]=wlan0.1
bssid[2]=02:9a:96:fb:59:12
ssid[2]=wave_11
num_sta[2]=0
bss[3]=wlan0.2
bssid[3]=02:9a:96:fb:59:13
ssid[3]=wave_12
num_sta[3]=0
bss[4]=wlan0.3
bssid[4]=02:9a:96:fb:59:14
ssid[4]=wave_13
num_sta[4]=0
```

 Fields `bssid` and `ssid` where already present in the `VAPElement` struct and then I just added the new `bss` field.